File: python/chutenumero.py
from random import randint

NUMERO=5
RANDOMNUMERO=randint(0, 10)
if RANDOMNUMERO == NUMERO:
    print('Acertou!')
else:
    if NUMERO > RANDOMNUMERO:
        print('Chutou Alto!')
    else:
        print('Chutou Baixo!')

File: javascript/chutenumero.js
const numero = 5
const randomnumero = Math.floor(Math.random() * 10)
if(randomnumero == numero) {
    console.log("acertou!");
}
if(randomnumero > numero) {
    console.log("Chutou baixo!")
} else {
    console.log("Chutou alto!")
}